typegate/tests/runtimes/prisma/full_prisma_mapping.py

```diff
@@ -79,4 +80,28 @@ def prisma(g: Graph):
                 "where": {"id": 10007},
             }
         ),
+        testExecuteRaw=db.execute(
+            'UPDATE "Post" SET title = $1 WHERE title like $2',
```

Comment:
I believe named arg are worth supporting to avoid ordering issue `${test}`, then you can delegate the replacement by other characters to the rust and then the final replacement to Prisma